src/renderer/gdi/paint.cpp

```diff
@@ -685,7 +685,7 @@ try
     const auto srcWidth = imageSlice.PixelWidth();
     const auto srcHeight = srcCellSize.height;
     const auto dstWidth = srcWidth * dstCellSize.width / srcCellSize.width;
-    const auto dstHeight = srcHeight * dstCellSize.height / srcCellSize.height;
+    const auto dstHeight = dstCellSize.height;

```

Comment:
This is just a little optimization that I noticed when investigating the rounding problems. The `srcHeight` is always `srcCellSize.height`, so the `dstHeight` should always be `dstCellSize.height`.